beeline/propagation/test_honeycomb.py

```diff
@@ -21,6 +22,7 @@ def test_roundtrip(self):
 
     def test_roundtrip_with_dataset(self):
         '''Verify that we can successfully roundtrip (marshal and unmarshal)'''
+        beeline.propagation.propagate_dataset = True
```

Comment:
ask: can we add tests for disabled propagation

Reply:
https://github.com/honeycombio/beeline-python/pull/213/commits/6f876be7ec211e43af08da3b1f042642ba9b72c6